CHANGELOG-lint-updates.md

```diff
@@ -0,0 +1,3 @@
+- Update project ESLint configuration and dependencies.
```

Comment:
This seems to have escaped the merge.

Reply:
👀 on it, I think this must have gotten propagated from an earlier merge